cli.js

```diff
@@ -284,6 +284,23 @@ Usage: $0 <command> [options]` // eslint-disable-line comma-dangle
     }
     await handleCommand(argv, statsHandler);
   })
+  .command('show', 'Show information about your Binaris account', (yargs0) => {
+    yargs0
+      .usage('Usage: $0 show [options]')
+      .option('account_id', {
+        describe: 'Show configured account ID',
+        requiresArg: false,
+      })
+      .option('api_key', {
+        describe: 'Show configured account API key',
+        requiresArg: false,
+      })
+      .check(
+        (argv) => {
+          if (!argv.account_id && !argv.api_key) throw new Error('Must specify what to show.');
```

Comment:
This can be done in a much cleaner way (I believe) using the `demand` option. You can simply use `demand: 1` which will force 1 arg.